app/templates/HOLD-landing-page.hbs

```diff
@@ -12,7 +12,7 @@
         <div class='container'>
             <div class='row'>
                 <div class='col-sm-4 col-sm-push-8'>
-                    {{dashboard-sidebar providers=model}}
+                    {{dashboardSidebar providers=model}}
```

Comment:
You got an actual component by mistake!

Though we don't use this template, I think you could just delete it. If we ever want it back, it's forever in git.

Reply:
👀 oops!!